src/main/exit-app.ts

```diff
@@ -26,19 +26,11 @@ import { ClusterManager } from "./cluster-manager";
 import logger from "./logger";
 
 export function exitApp() {
-  console.log("before windowManager");
-  const windowManager = WindowManager.getInstance(false);
-
-  console.log("before clusterManager");
-  const clusterManager = ClusterManager.getInstance(false);
-
-  console.log("after clusterManager");
-
   appEventBus.emit({ name: "service", action: "close" });
-  windowManager?.hide();
-  clusterManager?.stop();
+
+  WindowManager.resetInstance();
```

Comment:
should hide() still be called, or is it moot since we're exiting?

Reply:
I would say moot